### PR TITLE
Completed javascript-carousel Exercise.

### PR DESCRIPTION
**Pull request by annuhpark**
https://user-images.githubusercontent.com/69396309/163256704-d8be986d-71ad-419a-958b-acbfcb842d34.mov

### CODE REVIEW

**Comment by annuhpark** [comment]
Also showing setInterval();

https://user-images.githubusercontent.com/69396309/163256955-e6250b55-af3a-4d08-bbcc-e6ee3333a3dc.mov

**Review by TimelikeClosure — CHANGES_REQUESTED** [review]
- Whenever the slide changes, the filled-in dot should change to match.
- When the user manually switches slides, the interval should start over.

**Comment by annuhpark** [comment]
https://user-images.githubusercontent.com/69396309/163261852-8881e81f-3ffd-480f-b5b7-6400263deb1f.mov

File: javascript-carousel/main.js [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+function changeImage() {
+  $image.setAttribute('src', 'images/' + imageSrcValues[intervalIndex]);
+  matchIntervalToCircle();
+  intervalIndex++;
+  if (intervalIndex === 5) {
+    intervalIndex = 0;
+  }
+}
+
+setInterval(changeImage, 3000);
```
Comment (TimelikeClosure) [inline javascript-carousel/main.js:19]: The countdown to automatically switch images should start over when the user manually changes it:

---

image

**Review by TimelikeClosure — CHANGES_REQUESTED** [review]

**Review by TimelikeClosure — APPROVED** [review]

**Pull request merged**